docs/_static/custom.css

```diff
@@ -0,0 +1,10 @@
+.dropdown-related-projects ul.dropdown-menu ,
+.dropdown-versions ul.dropdown-menu {
+    /* Menus of launch buttons are normally shifted by -75%.
+       This makes no sense for buttons with text. */
+    transform:translateX(0);
+}
```

Comment:
The `>>>` are no longer orange in the examples extracted from the docstrings :-( 
![Screenshot at 2023-03-13 15-06-06](https://user-images.githubusercontent.com/39047984/224725573-0629acb6-3a91-44e7-a1b2-1829ed1a9ed4.png)

Reply:
The highlight theme was changed to be more friendly to colourblind people and people who struggle with low contrast. But I don't like it. The colours are barely visible now, it's all just a blend of dirty greys. There are options to change it, but IMHO, they are all bad.

Reply:
> it's all just a blend of dirty greys

agreed